src/commands/kv/mod.rs

```diff
@@ -0,0 +1,66 @@
+use cloudflare::auth::Credentials;
+use cloudflare::response::APIFailure;
+use cloudflare::response::APIResponse;
+use cloudflare::response::APIResult;
+use cloudflare::HTTPAPIClient;
+
+use crate::settings;
+use crate::terminal::message;
+
+mod create_namespace;
+
+pub use create_namespace::create_namespace;
+
+fn api_client() -> Result<HTTPAPIClient, failure::Error> {
+    let user = settings::global_user::GlobalUser::new()?;
+
+    Ok(HTTPAPIClient::new(Credentials::from(user)))
+}
+
+fn account_id() -> Result<String, failure::Error> {
+    let project = settings::project::Project::new()?;
+    // we need to be certain that account id is present to make kv calls
+    if project.account_id.is_empty() {
+        panic!("Your wrangler.toml is missing the account_id field which is required to create KV namespaces!");
```

Comment:
any reason to panic here and not just return an error?

Reply:
nope. should clean that up.

Reply:
#406